src/main/java/ama/post/CommentPostRepository.java

```diff
@@ -9,7 +9,7 @@
  * Created by stephanernst on 3/20/2017.
  */
 @Transactional
-public interface CommentPostRepository extends PostRepository {
+public interface CommentPostRepository extends PostRepository<CommentPost> {
```

Comment:
It was done in SubmissionPostRespository and I assumed it had to be for this repository as well. It made retrieving the saved comment work.